src/components/UI/Routines.js.jsx

```diff
@@ -1,12 +1,30 @@
 import React, { Component } from 'react'
+import { Link } from 'react-router'
 import { Col } from 'react-bootstrap'
+import Routine from './Routine.js.jsx'
 import './styles/Routines.css'
 
 class Routines extends Component {
+  componentDidMount() {
+    this.getRoutines(8)
+  }
+
+  getRoutines = (num) => {
```

Comment:
The num is arbitrary, I'm just passing the same number of workouts that's on the wireframes. All this method returns is a list of links. The links themselves direct to the individual workout pages.